spec/system/photos_spec.rb

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+require 'rails_helper'
+
+RSpec.describe 'PhotosController', type: :system do
+
+  before do
+    FileUtils.cp_r("#{Dir.pwd}/spec/fixtures/files/backup_photos/photos", "#{Dir.pwd}/public")
+
+    visit ENV['LINK']
+  end
+
+  after(:all) do
+    FileUtils.cp_r("#{Dir.pwd}/spec/fixtures/files/backup_photos/photos", "#{Dir.pwd}/public")
+  end
+
+  
+  def get_filename
+    @get_photo_filename = page.body.scan(/<img src="\/photos\/(.+)">/)
+    @get_photo_filename.flatten!
+  end
+
+  def get_href
+    @get_delete_href = page.body.scan(/href="(.+)">delete/)
+    @get_delete_href.flatten!
+  end
+
+
+  it "webページ上にMy Photosという文字列が表示されている。" do
+    expect(page).to have_content 'My Photos'
+  end
+
+
+  it "webページ上に削除対象の画像（webページ上で先頭にある画像）が存在する。" do
+    get_filename
+    expect(page).to have_selector("img[src$='#{@get_photo_filename.at(0)}']")
```

Comment:
```suggestion
    expect(page).to have_selector("img[src$='#{@get_photo_filename.first}']")
```

先頭を取得する場合は`first`を使うのが多い気がします